libkbfs/keybase_service_base.go

```diff
@@ -437,19 +445,68 @@ func (k *KeybaseServiceBase) LoadUserPlusKeys(ctx context.Context,
 	return k.processUserPlusKeys(res)
 }
 
+var allowedLoadTeamRoles = map[keybase1.TeamRole]bool{
+	keybase1.TeamRole_NONE:   true,
```

Comment:
When would `TeamRole_NONE` be used? The code below seems to suggest it's the same as `TeamRole_READER`?